server/lobbyconnection.py

```diff
@@ -888,6 +888,31 @@ def launch_game(self, game, port, is_host=False, use_map=None):
             else:
                 raise ValueError('invalid type argument')
 
+    async def command_invite_to_party(self, message):
+        invited_player = self.player_service.get_player(message["invited_player_id"])
+        if invited_player is None:
+            raise ClientError("The invited player doesn't exist", recoverable=True)
+        else:
```

Comment:
The `else` is unnecessary since raising an exception will exit the function.

Reply:
true, I'm busy grasping python and the server structure, would have never done such a thing in java ;)